app/page.tsx

```diff
@@ -68,11 +69,68 @@ function AboutUsOverview() {
   );
 }
 
+function ProgramsOverview() {
+
+  const backContent_1 = [
```

Comment:
i had to add a key prop element into these arrays but not sure if it is necessary @sophiac6410 @angewen?